localization/ndt_scan_matcher/include/ndt_scan_matcher/ndt_scan_matcher_core.hpp

```diff
@@ -152,6 +152,8 @@ class NDTScanMatcher : public rclcpp::Node
     initial_pose_with_covariance_pub_;
   rclcpp::Publisher<geometry_msgs::msg::PoseArray>::SharedPtr multi_ndt_pose_pub_;
   rclcpp::Publisher<geometry_msgs::msg::PoseArray>::SharedPtr multi_initial_pose_pub_;
+  rclcpp::Publisher<tier4_debug_msgs::msg::Float32Stamped>::SharedPtr ndt_dev_x_pub_;
+  rclcpp::Publisher<tier4_debug_msgs::msg::Float32Stamped>::SharedPtr ndt_dev_y_pub_;
```

Comment:
These publishers may not be necessary, as these values ​​can be calculated from the output of the covariance matrix. I would like to hear other people's opinions on this.

Reply:
I also think these publishers are unnecessary. It might be useful for debugging, but for practical purposes, I think pose_with_covariance is sufficient.

Reply:
I also think these publishers are unnecessary. 
It might be useful for debugging, but for practical usages, I think pose_with_covariance is sufficient.

Reply:
OK! Thanks for pointing that out!